### ISSUE TITLE
Only show status bar on command

### ISSUE THREAD

**Issue by zbraniecki** [zellij-org/zellij#1771]
Similarly to how GUI menu bar is now often hidden (on Linux and Windows at least) until you click `alt`, it would be great to have an option to have the status bar hidden until I enable `ctrl+t` or `ctrl+p`.

**Comment by imsnif** [comment]
Agreed - this unfortunately depends on this: https://github.com/zellij-org/zellij/issues/742
Which depends on a bit of other stuff. I'm unraveling this pile of tech-debt slowly but surely!
